app/config/locale/translations/es.json

```diff
@@ -3,30 +3,30 @@
     "settings.locale": "es",
     "settings.direction": "ltr",
     "emails.sender": "Equipo %s",
-    "emails.verification.subject": "",
-    "emails.verification.hello": "",
-    "emails.verification.body": "",
-    "emails.verification.footer": "",
-    "emails.verification.thanks": "",
-    "emails.verification.signature": "",
-    "emails.magicSession.subject": "",
-    "emails.magicSession.hello": "",
-    "emails.magicSession.body": "",
-    "emails.magicSession.footer": "",
-    "emails.magicSession.thanks": "",
-    "emails.magicSession.signature": "",
-    "emails.recovery.subject": "",
-    "emails.recovery.hello": "",
-    "emails.recovery.body": "",
-    "emails.recovery.footer": "",
-    "emails.recovery.thanks": "",
-    "emails.recovery.signature": "",
-    "emails.invitation.subject": "",
-    "emails.invitation.hello": "",
-    "emails.invitation.body": "",
-    "emails.invitation.footer": "",
-    "emails.invitation.thanks": "",
-    "emails.invitation.signature": "",
+    "emails.verification.subject": "Verificación de cuenta",
+    "emails.verification.hello": "Hola {{name}}",
+    "emails.verification.body": "Haz clic en este enlace para verificar tu correo.",
+    "emails.verification.footer": "Si no has solicitado verificar este correo, puedes ignorar este mensaje.",
+    "emails.verification.thanks": "Gracias",
+    "emails.verification.signature": "Equipo de {{project}}",
+    "emails.magicSession.subject": "Inicio de Sesión",
+    "emails.magicSession.hello": "Hola,",
+    "emails.magicSession.body": "Haz clic en este enlace para iniciar sesión.",
+    "emails.magicSession.footer": "Si no has solicitado ingresar usando este correo, puedes ignorar este mensaje.",
+    "emails.magicSession.thanks": "Gracias",
+    "emails.magicSession.signature": "Equipo de {{project}}",
+    "emails.recovery.subject": "Restablecer contraseña",
+    "emails.recovery.hello": "Hola {{name}}",
+    "emails.recovery.body": "Haz clic en este enlace para restablecer la contraseña de tu proyecto {{project}}.",
+    "emails.recovery.footer": "Si no has solicitado restablecer la contraseña, puedes ignorar este mensaje.",
+    "emails.recovery.thanks": "Gracias",
+    "emails.recovery.signature": "Equipo de {{project}}",
+    "emails.invitation.subject": "Invitacion al Equipo %s en %s",
```

Comment:
Hi, 
'Invitacion' => you need the accent on the 'o'. 'Invitación'.
Equipo => should be written in lower case. 'equipo'

Reply:
Hey, thanks a lot! Already fix that